semgrep_metrics.atd

```diff
@@ -204,13 +204,22 @@ type pro_features = {
      * Since semgrep 1.46
      *)
     ?diffDepth <ocaml mutable>: int option;
+
     (* The number of scanned files per language for inter-file diff scan mode.
      * This number represents the count of changed files and their limited
      * dependencies.
      * Since semgrep 1.70
      *)
     ?numInterfileDiffScanned <ocaml mutable>: (string (* lang *) * int) list
                                               <json repr="object"> option;
+
+    (* The number of interface fields resolved during the Naming_SAST phases.
+     * This is the total count from the naming resolution of all field accesses
+     * in the input source code.
+     * Since semgrep 1.91
+     *)
+    ?numInterfaceVarResolved <ocaml mutable>: int option;
```

Comment:
I don't think we want to collect such metrics. If you're interested in improving our name resolution, just do some benchmarks and extend the metrics in Stat_code.ml (which is used in Graph_code_SIG_SAST.ml for example).

Let's say we collect those metrics, what will you do based on the value?